src/player.cpp

```diff
@@ -3073,15 +3058,30 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
             fold_and_print( w_info, 0, 1, FULL_SCREEN_WIDTH - 2, c_magenta, s );
             wrefresh(w_info);
 
-
             action = ctxt.handle_input();
-            if (action == "DOWN") {
-                if (line < 11) {
-                    line++;
+            if ( action == "DOWN" )
+            {
+                if ( line < num_bp - 1 ) {
+                    if (
```

Comment:
Don't break up conditionals like this.  As much of the conditional as will fit on the initial line with the ```if(```should go there, and the ending ```) {``` should be on the same line as the last part of the conditional.